android/src/main/java/com/heapanalytics/reactnative/HeapInitializationProvider.java

```diff
@@ -0,0 +1,42 @@
+package com.heapanalytics.reactnative;
```

Comment:
Hey @kamalmarhubi - any thoughts about changing the package name to match the company name (i.e. drop the analytics)?  Are we locked into that for weird reasons?